ash-window/src/lib.rs

```diff
@@ -171,3 +171,102 @@ pub fn enumerate_required_extensions(
 
     Ok(extensions)
 }
+
+/// Query whether a `queue_family` of the given `physical_device` supports presenting to any
+/// surface that might be created. This function can be used to find a suitable
+/// [`vk::PhysicalDevice`] and queue family for rendering before a single surface is created.
+///
+/// This function can be a more useful alternative for [`vkGetPhysicalDeviceSurfaceSupportKHR`],
```

Comment:
I meant to just have:
```suggestion
/// This function can be a more useful alternative for [`khr::Surface::get_physical_device_surface_support()`],
```

We don't generally use Vulkan symbol naming in Ash.